src/models/aero_state.jl

```diff
@@ -75,8 +75,10 @@ function AeroState(tas::Number, alpha::Number, beta::Number, height::Number)
     AeroState(alpha, beta, 0, tas, eas, cas, ias, qinf, mach)
 end
 
-
-function state_aerostate(pos, att, tas, alpha, beta, env=DefaultEnvironment(),
+"""
```

Comment:
Could you improve this just a little bit and follow the style guide? It will be great for doc https://alexs12.github.io/FlightMechanics.jl/latest/

Reply:
what are you suggesting?  should just delete DefaultEnviornment() or you would like to actually create default environment function.

Reply:
I think it is a better idea to call this function with a concrete environment. We can also have a `DefaultEnvironment` that could be used when calling the function or anywhere else.

Reply:
in my opinion DefaultEnvironment is not a good idea. i.e. if we use pos=[0,0,0] for Default Environment and input pos=[0, 0, 1000] in other function it will give wrong answer. so it is best not to create Default Environment.